pept/scanners/parallel_screens/parallel_screens.py

```diff
@@ -28,134 +28,141 @@
+		An overlap of 0 means consecutive samples, while an overlap
+		of (`sample_size` - 1) means incrementing the samples by one.
+		A negative overlap means skipping values between samples. An
+		error is raised if `overlap` is larger than or equal to
+		`sample_size`. (Default is 0)
+	separation : float, optional
+		The separation (in *mm*) between the two PEPT screens corresponding
+		to the `z` coordinate of the second point defining each line.
+		The attribute `line_data`, with each row being
+		`[time, x1, y1, z1, x2, y2, z2]`, will have `z1 = 0` and
+		`z2 = separation`. (Default is 712)
+	skiprows : int, optional
+		The number of rows to skip from the beginning of the data file.
+		Useful when the data file includes a header of text that should
+		be skipped. (Default is 0)
+	max_rows : int, optional
+		The maximum number of rows that will be read from the data file.
+		(Default is `None`)
+	verbose : bool, optional
+		An option that enables printing the time taken for the
+		initialisation of an instance of the class. Useful when
+		reading large files (10gb files for PEPT data is not unheard
+		of). (Default is True)
+
+	Attributes
+	----------
+	line_data : (N, 7) numpy.ndarray
+		An (N, 7) numpy array that stores the PEPT LoRs as time and
+		cartesian (3D) coordinates of two points defining a line, **in mm**.
+		Each row is then `[time, x1, y1, z1, x2, y2, z2]`.
+	sample_size : int
+		An `int` that defines the number of lines that should be
+		returned when iterating over `line_data`. (Default is 200)
+	overlap : int
+		An `int` that defines the overlap between two consecutive
+		samples that are returned when iterating over `line_data`.
+		An overlap of 0 means consecutive samples, while an overlap
+		of (`sample_size` - 1) means incrementing the samples by one.
+		A negative overlap means skipping values between samples. It
+		has to be smaller than `sample_size`. (Default is 0)
+	number_of_lines : int
+		An `int` that corresponds to len(`line_data`), or the number of
+		LoRs stored by `line_data`.
+
+	Raises
+	------
+	ValueError
+		If `overlap` >= `sample_size`. Overlap has to be smaller than
+		`sample_size`. Note that it can also be negative.
+	ValueError
+		If the data file does not have (N, 5) shape.
+
+	Notes
+	-----
+	The class saves `line_data` as a **contiguous** numpy array for
+	efficient access in C functions. It should not be changed after
+	instantiating the class.
+
+	'''
+
+	def __init__(self,
+				 data_file,
+				 separation=None,
+				 sample_size = 200,
+				 overlap = 0,
+				 skiprows = 0,
+				 max_rows = None,
+				 verbose = True,
+				 saveas = None):
+
+		if verbose:
+			start = time.time()
+
+		# Check file extension
+
+		if data_file[-5:-2] == '.da':
+			print("Converting binary file...")
+			line_data = read_adac_binary(data_file, saveas, separation)
+			line_data = np.array(line_data)
+		else:
+			print("Assuming old csv file...")
+			# Read the LoR data from `data_file`
```

Comment:
Only print stuff out if the "verbose" option is True. Otherwise, we should have silent functions.